synapse/python_dependencies.py

```diff
@@ -150,7 +163,7 @@ def message(self):
     @property
     def dependencies(self):
         for i in self.args[0]:
-            yield "'" + i + "'"
+            yield '"' + i + '"'
```

Comment:
this line here is the actual fix - the rest of it is just to make sure we don't do the same thing again with ".